builds/release/release-core-images.yaml

```diff
@@ -0,0 +1,540 @@
+      displayName: Filter image list for core images
+      name: filter
+
+    - task: AzureKeyVault@1
+      displayName: Get secrets
+      condition: and(succeeded(), eq(variables['filter.continue'], 'true'))
+      inputs:
+        azureSubscription: $(az.subscription)
+        keyVaultName: $(kv.name)
+        secretsFilter: TestGitHubAccessToken
+
+    - script: |
+        #!/bin/bash
+        set -euo pipefail
+
+        . scripts/linux/release-tools.sh
+
+        GIT_EMAIL='$(service-account.email)' \
+        BRANCH="${BUILD_SOURCEBRANCH#refs/heads/}" \
+        make_project_release_commit_for_core_image_refresh
+        get_project_release_info
+
+        changelog=$(echo "$OUTPUTS" | jq -rc '.changelog')
+        next=$(echo "$OUTPUTS" | jq -rc '.version')
+        prev=$(echo "$OUTPUTS" | jq -rc '.previous_version')
+        diag_version=$(echo "$OUTPUTS" | jq -rc '.diagnostics_version')
+        tags=$(echo "$OUTPUTS" | jq -rc '.tags')
+
+        echo "##vso[task.setvariable variable=changelog;isOutput=true]$changelog"
+        echo "##vso[task.setvariable variable=version;isOutput=true]$next"
+        echo "##vso[task.setvariable variable=previous_version;isOutput=true]$prev"
+        echo "##vso[task.setvariable variable=diagnostics_version;isOutput=true]$diag_version"
+        echo "##vso[task.setvariable variable=tags;isOutput=true]$tags"
+      displayName: Create a release commit
+      name: commit
+      condition: and(succeeded(), eq(variables['filter.continue'], 'true'))
+      env:
+        GITHUB_TOKEN: "$(TestGitHubAccessToken)"
+
+  - job: ManualTrigger
+    displayName: Manually trigger pipeline to update images
+    condition: eq(variables['Build.Reason'], 'Manual')
+
+    steps:
+    - checkout: self
+      fetchDepth: 0
+
+    - script: |
+        # Create a filter.continue task variable to match the PipelineTrigger job, but hard-code it
+        # to true so the pipeline will always run subsequent stages in the manual case.
+        echo "##vso[task.setvariable variable=continue;isOutput=true]true"
+      name: filter
+
+    - script: |
+        #!/bin/bash
+        set -euo pipefail
+        
+        # this script assumes the following steps have already been taken:
+        # - a release commit has been pushed to the project repo, and is reachable from this run's
+        #   source commit
+        # - the release commit was tagged with the new version
+        # - the project repo's CHANGELOG.md has been updated with information about the new release
+
+        . scripts/linux/release-tools.sh
+        get_project_release_info
+
+        changelog=$(echo "$OUTPUTS" | jq -rc '.changelog')
+        next=$(echo "$OUTPUTS" | jq -rc '.version')
+        prev=$(echo "$OUTPUTS" | jq -rc '.previous_version')
+        diag_version=$(echo "$OUTPUTS" | jq -rc '.diagnostics_version')
+        tags=$(echo "$OUTPUTS" | jq -rc '.tags')
+
+        echo "##vso[task.setvariable variable=changelog;isOutput=true]$changelog"
```

Comment:
Won't this already be set from previous step?
https://github.com/Azure/iotedge/pull/6746/files#diff-b64f3d02f0615edcd5ab6f97f28f01823a4131cdb85f8b87e66bc9a27c307900R104

Reply:
No, the step you linked is in the `PipelineTrigger` job, which only runs when the pipeline is automatically triggered after detect-image-updates completes. This step is in the `ManualTrigger` job, which is analagous to the `PipelineTrigger` job, but for manually triggered builds.

Reply:
I can add a high-level explanation about how the `PrepareRelease` stage (to which both these jobs belong) is stuctured at the top of the stage.

Reply:
Ah I was looking at the condition for the step and not the job. I get it.

Reply:
Added a comment to the top `PrepareRelease`.